### PR TITLE
Feature: Hotkeys hint

### PR DESCRIPTION
**Pull request by maxplt**
# Jira Issue: [IP-198](https://hedvig.atlassian.net/jira/software/projects/IP/boards/19?selectedIssue=IP-198)

## What?
Make a new view of hotkey hints.
It's a draft because all hints at the moment for the Questions page and Templates modal, but navigation in it is still in work. 

## Why?
To let know what shortcuts press

## Video

https://user-images.githubusercontent.com/89198006/164611944-98c066cd-4665-49ea-9454-b25379bfba6a.mov

## Optional checklist
- [ ] Updated `src/changelog.ts`
- [ ] Codescouted
- [ ] Unit tests written
- [ ] Tested locally

### CODE REVIEW

File: src/portals/hope/features/tasks/CheckInMessage.tsx [hunk 1]
```diff
@@ -58,22 +60,25 @@ export const CheckInMessage: React.FC = () => {
...
         <Spacing top="small" />
         <Flex align="center">
-          <Button
-            size="medium"
-            onClick={() => {
-              if (markets.length === 0) {
-                setShowModal(true)
-                return
-              }
+          <HotkeyHint text="check in" keys={[Keys.Option, Keys.Enter]}>
```
Comment (vonElfvin) [inline src/portals/hope/features/tasks/CheckInMessage.tsx:63]: I think we should start these with a capital letter :)
```suggestion
          <HotkeyHint text="Check in" keys={[Keys.Option, Keys.Enter]}>
```

File: src/portals/hope/pages/tasks/TasksPage.tsx [hunk 2]
```diff
@@ -185,7 +186,7 @@ const FilterBarItem = styled(motion.button)`
   border-radius: 0.25rem;
   border: none;
   cursor: pointer;
-  margin: 1.8rem 2rem;
+  /* margin: 1.8rem 2rem; */
```
Comment (vonElfvin) [inline src/portals/hope/pages/tasks/TasksPage.tsx:189]: Oupsie

**Review by vonElfvin — COMMENTED** [review]
Nice! Would be nice if we added a "toast" with the exact same text from the bottom in case someone still clicks as well 🔥 

Like

"Next time, use Option + Enter to check in 😎"

**Comment by maxplt** [comment]
## Toast hint:

https://user-images.githubusercontent.com/89198006/165034270-7df484d1-9cbb-4504-ae2f-63e87cfa5f8c.mov

File: src/portals/hope/features/tasks/list-items/TaskListItem.tsx [hunk 3]
```diff
@@ -30,9 +30,9 @@ const ListItem = styled(motion.li)<{ selected?: boolean }>`
     }
     padding-right: 2rem;
 
-    white-space: nowrap;
-    overflow: hidden;
-    text-overflow: ellipsis;
+    /* white-space: nowrap; */
+    /* overflow: hidden; */
+    /* text-overflow: ellipsis; */
```
Comment (rasmusguterstam) [inline src/portals/hope/features/tasks/list-items/TaskListItem.tsx:35]: 🤔

**Review by rasmusguterstam — APPROVED** [review]
Wow, really nice!

I'm not really sure what I think about the toast though haha, it's super annoying _but_ I also like that it's the point, to promote keyboard usage. This might actually be the way to go, but we will have to make sure that hotkeys work flawlessly, otherwise this will be really painful 💀 

I think we'll have to make a proper revisit of the command line / `registerActions` feature, since it's quite buggy / unstable in its current state.

**Review by rasmusguterstam — APPROVED** [review]

**Pull request merged**